_posts/2013-08-10-companies.markdown

```diff
@@ -95,35 +96,38 @@ layout: sidebar
-| `name`                    | name of this company.
-| `trade_name`              | trade name of this company. This is an optional field.
-| `ein`                     | employer identification number for this company.
-| `entity_type`             | type of company.
-| `locations`               | array of addresses for all known locations of this company.
-| `id`                      | the unique identifier of this location in the Gusto system.
-| `street_1`                | first line of this address' street.
-| `street_2`                | second line of this address' street.
-| `city`                    | name of address' city.
-| `state`                   | two letter state abbreviation.
-| `zip`                     | five digit zipcode. Returned as a string to maintain leading zeros.
-| `compensations`           | all available company-wide compensations. The above compensations are the default, built-in compensations.
-| `hourly`                  | available hourly compensations
-| `name`                    | name of the compensation
-| `multiple`                | amount multiplied by the base rate of a given job to calculate total compensation per hour worked. For example, if an employee at this company makes $17.00/hour as Kindergarten Cop and works 10 hours of Overtime (which has a multiple of 1.5), he would expect receive $150.
-| `fixed`                   | available fixed compensations
-| `name`                    | name of the compensation
-| `paid_time_off`           | available types of paid time off
-| `name`                    | name of the time off
-| `first_name`              | primary signatory's first name
-| `middle_initial`          | primary signatory's middle initial
-| `last_name`               | primary signatory's last name
-| `phone`                   | primary signatory's phone number
-| `email`                   | primary signatory's email
-| `home_address`            | primary signatory's home address
-| `first_name`              | primary payroll admin's first name
-| `last_name`               | primary payroll admin's last name
-| `phone`                   | primary payroll admin's phone number
-| `email`                   | primary payroll admin's email
+| Attribute                          | Type                           | Description
+| :----------                        |:-------------                  |:-------------
+| `id`                               | Integer                        | The unique identifier of this company in the Gusto system.
+| `name`                             | String                         | The name of this company.
+| `trade_name`                       | String                         | The trade name of this company. This is an optional field.
+| `ein`                              | String                         | The employer identification number for this company.
+| `entity_type`                      | String                         | The type of the company (e.g. "LLC").
```

Comment:
Can we list all of the possible values for this? If it isn't easy, we can skip it. I was mostly curious if we could be more illustrative about return values. :)

Reply:
There's quite a few values so I didn't list them all, but added the top 5 according to Redash in the most recent commit.